manifests/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -23,8 +23,24 @@ spec:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
     spec:
       serviceAccountName: olm-operator-serviceaccount
+      volumes:
+        - name: srv-cert
+          secret:
+            secretName: olm-operator-serving-cert
+        - name: profile-collector-cert
+          secret:
+            secretName: olm-operator-serving-cert
       containers:
         - name: olm-operator
+          volumeMounts:
+            - name: srv-cert
+              mountPath: "/srv-cert"
+              readOnly: true
+            - name: profile-collector-cert
+              mountPath: "/profile-collector-cert"
+              readOnly: true
+            - mountPath: /var/run/secrets/serving-cert
+              name: serving-cert
```

Comment:
```suggestion
            - mountPath: /var/run/secrets/serving-cert
              name: serving-cert
```

Reply:
Oh - this should actually be removed from `scripts/olm-deployment.patch.yaml`